devel/libical/Portfile

```diff
@@ -1,22 +1,19 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
-# $Id$
 
 PortSystem          1.0
-PortGroup           cmake 1.0
+PortGroup           cmake 1.1
 PortGroup           github 1.0
 
-cmake.out_of_source yes
-
 name                libical
 
 if {${subport} eq "${name}"} {
-	github.setup    libical libical 1.0.1 v
-	conflicts       libical-devel
+    github.setup    libical libical 2.0.0 v
+    conflicts       libical-devel
```

Comment:
There is no difference whether you do it to master or to another branch, you need to "rebase" or reapply the same changes in any case. But I decided to commit to master because the changes were straightforward and there was no need to keep them in this PR for discussion, it makes the PR cleaner this way.